{"name": "测试数据：【 wintest3，False，3006，该用户不允许删除 】", "status": "broken", "statusDetails": {"message": "AttributeError: 'Response' object has no attribute 'response'", "trace": "tp = <class 'AttributeError'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\n\n.venv/lib/python3.9/site-packages/six.py:723: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\n    @pytest.fixture(scope=\"session\")\n    def login_fixture():\n        username = base_data[\"init_admin_user\"][\"username\"]\n        password = base_data[\"init_admin_user\"][\"password\"]\n        header = {\n            \"Content-Type\": \"application/x-www-form-urlencoded\"\n        }\n        payload = {\n            \"username\": username,\n            \"password\": password\n        }\n        loginInfo = user.login(data=payload, headers=header)\n        step_login(username, password)\n        # yield loginInfo.json()\n>       yield loginInfo.response.json()\nE       AttributeError: 'Response' object has no attribute 'response'\n\ntestcases/conftest.py:57: AttributeError"}, "description": "该用例是针对获取用户删除接口的测试", "parameters": [{"name": "username", "value": "'wintest3'"}, {"name": "except_result", "value": "False"}, {"name": "except_code", "value": "3006"}, {"name": "except_msg", "value": "'该用户不允许删除'"}], "start": 1744990584476, "stop": 1744990584476, "uuid": "c3bf3d6b-36d5-4a4c-89d1-f71a29eaa1e1", "historyId": "6a8136563fec3910840f1add5a7a645c", "testCaseId": "f140003f8969cec64326a77ad093297e", "fullName": "testcases.api_test.test_05_delete_user.TestUserDelete#test_delete_user", "labels": [{"name": "severity", "value": "normal"}, {"name": "story", "value": "用例--删除用户信息"}, {"name": "feature", "value": "用户删除模块"}, {"name": "epic", "value": "针对单个接口的测试"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "testcases.api_test"}, {"name": "suite", "value": "test_05_delete_user"}, {"name": "subSuite", "value": "TestUserDelete"}, {"name": "host", "value": "192.168.10.5"}, {"name": "thread", "value": "42691-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.test_05_delete_user"}], "links": [{"type": "test_case", "url": "https://blog.csdn.net/", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://blog.csdn.net/", "name": "点击，跳转到对应BUG的链接地址"}]}